content/slapen.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Slapen
+date: 2017-03-23
+publishdate: 2017-03-24
+menu:
+  main:
+    weight: 3
+---
+
+Slapen doen wij niet in ons eigen bedje maar in het Stay Okay Hotel in Arnhem. Wij willen jullie van harte uitnodigen om hetzelfde te doen. Het is mogelijk om te blijven slapen vanaf &euro; 20,50. Zo, kunnen we samen genieten van een (kater)ontbijt.
```

Comment:
Wilden we dit niet samen voegen met de pagina "Praktisch"?

Reply:
Gedaan